src/main/java/com/minecolonies/coremod/colony/jobs/JobDeliveryman.java

```diff
@@ -254,7 +254,7 @@ public void finishRequest(final boolean successful)
         else if (request.getRequest() instanceof Delivery)
         {
             final List<IRequest<? extends Delivery>> taskList = getTaskListWithSameDestination((IRequest<? extends Delivery>) request);
-            for (int i = 0; i < Math.min(ongoingDeliveries, taskList.size()); i++)
+            for (int i = 0; i < Math.max(1, Math.min(ongoingDeliveries, taskList.size())); i++)
```

Comment:
there is math.clamp for limiting values both ways :D

Reply:
there is not